.gitlab/benchmark.yml

```diff
@@ -15,6 +15,8 @@
   - export BASELINE_DSD_VERSION="${BASE_DD_AGENT_VERSION}"
   - export COMPARISON_DSD_VERSION="${BASE_DD_AGENT_VERSION}"
   - export SOURCE_DSD_IMG="public.ecr.aws/datadog/dogstatsd:${BASE_DD_AGENT_VERSION}"
+  - export SOURCE_CHECK_AGENT_GO_IMG="${SMP_ECR_HOST}/08450328-agent:4c4f5e817c81a3622f1fc64959ab3dbea49c73da-7-full-amd64"
```

Comment:
This image is created and published from this branch https://github.com/DataDog/datadog-agent/pull/31571
Here is the GL job that publish it https://gitlab.ddbuild.io/DataDog/public-images/-/jobs/906265447